gensim/corpora/wikicorpus.py

```diff
@@ -380,6 +380,13 @@ def extract_pages(f, filter_namespaces=False):
                 if ns not in filter_namespaces:
                     text = None
 
+            if callable(filter_articles) and text:
```

Comment:
What if `filter_articles` isn't a callable? If I understand correctly, it should be a callable always, no need to check.

Reply:
That's just catching the default case where `filter_articles=None` - I didn't see the point of introducing a dummy callable that always returns true. I guess you could make the default `filter_articles=False` but that confuses the type semantics of `filter_articles` is it `bool` or a `callable`

Reply:
`is not None` preferred (since that's what we really want to test). But a dummy `filter_articles=gensim.utils.identity` is also OK.